cluster/images/kubemark/BUILD

```diff
@@ -4,7 +4,7 @@ load("@io_bazel_rules_docker//container:container.bzl", "container_image", "cont
 
 container_image(
```

Comment:
this should probably be converted to the `multi_arch_container` macro (similar to the one used in `cluster/images/hyperkube/BUILD`).